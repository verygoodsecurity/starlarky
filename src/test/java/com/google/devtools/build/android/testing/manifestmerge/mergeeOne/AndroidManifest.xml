<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
      package="com.google.android.apps.testapp2"
      android:versionCode="1"
      android:versionName="1.0">
  <permission android:name="com.google.android.apps.foo.C2D_MESSAGE"
     android:protectionLevel="signature" />
  <!-- minSdkVerion="3" to test that implicit permissions are not added. -->
  <uses-sdk android:minSdkVersion="3" />
  <uses-feature android:name="android.hardware.nfc" android:required="true" />
  <uses-permission android:name="android.permission.READ_LOGS" />
  <uses-permission android:name="android.permission.INTERNET" />
  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
  <!-- Comment for permission android.permission.GET_ACCOUNTS.
    This is just to make sure the comment is being merged correctly.
  -->
  <uses-permission android:name="android.permission.GET_ACCOUNTS" />
  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
  <application
      android:icon="@drawable/icon"
      android:label="@string/app_name"
      android:backupAgent="FooBar">
    <activity android:name=".ui.home.HomeActivity"
              android:label="@string/app_name" >
      <intent-filter>
        <action android:name="android.intent.action.MAIN"/>
        <category android:name="android.intent.category.LAUNCHER"/>
      </intent-filter>
    </activity>
    <activity android:name=".TestActivity2"></activity>
    <activity android:name=".PreviewActivity"></activity>
    <activity android:name=".ShowTextActivity" android:excludeFromRecents="true"></activity>
    <activity android:name=".ShowStringListActivity"
              android:excludeFromRecents="true"
              android:parentActivityName=".ui.home.HomeActivity">
    </activity>
    <service android:name=".TestService">
      <meta-data android:name="param" android:value="value"/>
    </service>
    <service android:name=".nfcevent.NfcEventService" />
    <receiver android:name=".ConnectivityReceiver" android:enabled="false" >
      <intent-filter>
        <action android:name="android.net.conn.CONNECTIVITY_CHANGE" />
      </intent-filter>
    </receiver>
    <activity-alias android:name="BarFoo" android:targetActivity=".FooBar" />
    <provider
      android:name="some.package.with.inner.class$AnInnerClass" />
    <provider
      android:name="${packageName}"
      android:authorities="${packageName}.${packageName}"
      android:exported="false" />
    <provider
      android:name="${packageName}.PlaceHolderProviderName"
      android:authorities="PlaceHolderProviderAuthorities.${packageName}"
      android:exported="false" />
    <activity
      android:name="activityPrefix.${packageName}.activitySuffix">
      <intent-filter>
        <action android:name="actionPrefix.${packageName}.actionSuffix" />
      </intent-filter>
    </activity>
  </application>
</manifest>
