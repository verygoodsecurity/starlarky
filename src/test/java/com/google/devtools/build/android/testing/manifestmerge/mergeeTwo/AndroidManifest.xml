<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
      package="com.google.android.apps.testapp3"
      android:versionCode="3"
      android:versionName="1.30">
  <uses-sdk android:minSdkVersion="14" />
  <uses-feature android:name="android.hardware.nfc" android:required="true" />
  <uses-permission android:name="android.permission.READ_LOGS" />
  <uses-permission android:name="android.permission.INTERNET" />
  <application>
    <activity android:name=".ui.home.HomeActivity"
              android:label="@string/app_name" >
      <intent-filter>
        <action android:name="android.intent.action.MAIN"/>
        <category android:name="android.intent.category.LAUNCHER"/>
      </intent-filter>
    </activity>
    <activity android:name="TestActivity"></activity>
    <service android:name=".TestService" />
    <receiver android:name=".ConnectivityReceiver" android:enabled="true" >
      <intent-filter>
        <action android:name="android.net.conn.CONNECTIVITY_CHANGER" />
      </intent-filter>
    </receiver>
  </application>
</manifest>
