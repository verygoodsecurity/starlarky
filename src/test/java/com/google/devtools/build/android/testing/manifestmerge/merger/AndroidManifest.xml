<?xml version="1.0" encoding="utf-8"?>
<manifest
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    package="com.google.android.apps.testapp"
    android:versionCode="70"
    android:versionName="1.0">
  <uses-sdk
    android:minSdkVersion="10"
    tools:overrideLibrary="com.google.android.apps.testapp3"/>
  <uses-feature android:name="android.hardware.nfc" android:required="true" />
  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
  <application
      android:icon="@drawable/icon"
      android:name="com.google.android.apps.testapp.TestApplication"
      android:theme="@style/Theme.Test"
      android:label="@string/app_name">
    <!--  START LIBRARIES (Maintain Alphabetic order) -->
    <!-- NFC extras -->
    <uses-library android:name="com.google.android.nfc_extras" android:required="false"/>
    <!--  END LIBRARIES -->
    <!--  START ACTIVITIES (Maintain Alphabetic order) -->
    <!-- Entry point activity - navigation and title bar. -->
    <activity
        android:name=".entrypoint.EntryPointActivityGroup"
        android:screenOrientation="portrait"
        android:launchMode="singleTop"/>
    <activity android:name=".ui.topup.TopUpActivity" />
    <service android:name=".nfcevent.NfcEventService" />
    <receiver
         android:name="com.receiver.TestReceiver"
         android:process="@string/receiver_service_name">
      <!-- Receive the actual message -->
       <intent-filter>
         <action
           android:name="android.intent.action.USER_PRESENT"/>
       </intent-filter>
    </receiver>
    <provider
        android:name=".dataaccess.persistence.ContentProvider"
        android:authorities="com.google.android.apps.testapp"
        android:exported="false" />
  </application>
</manifest>
