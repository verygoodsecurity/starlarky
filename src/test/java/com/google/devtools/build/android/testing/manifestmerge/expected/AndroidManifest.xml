<?xml version="1.0" encoding="utf-8"?>
<manifest
    xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.google.android.apps.testapp"
    android:versionCode="70"
    android:versionName="1.0" >
  <uses-sdk android:minSdkVersion="10" />
  <uses-feature android:name="android.hardware.nfc" android:required="true" />
  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
  <permission android:name="com.google.android.apps.foo.C2D_MESSAGE" android:protectionLevel="signature" />
  <android:uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
  <android:uses-permission android:name="android.permission.READ_PHONE_STATE" />
  <android:uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
  <application
      android:name="com.google.android.apps.testapp.TestApplication"
      android:backupAgent="com.google.android.apps.testapp2.FooBar"
      android:icon="@drawable/icon"
      android:label="@string/app_name"
      android:theme="@style/Theme.Test" >
    <!--  START LIBRARIES (Maintain Alphabetic order) -->
    <!-- NFC extras -->
    <uses-library android:name="com.google.android.nfc_extras" android:required="false" />
    <!--  END LIBRARIES -->
    <!--  START ACTIVITIES (Maintain Alphabetic order) -->
    <!-- Entry point activity - navigation and title bar. -->
    <activity
        android:name="com.google.android.apps.testapp.entrypoint.EntryPointActivityGroup"
        android:launchMode="singleTop"
        android:screenOrientation="portrait" />
    <activity android:name="com.google.android.apps.testapp.ui.topup.TopUpActivity" />
    <service android:name="com.google.android.apps.testapp.nfcevent.NfcEventService" />
    <receiver
         android:name="com.receiver.TestReceiver"
         android:process="@string/receiver_service_name" >
      <!-- Receive the actual message -->
       <intent-filter>
         <action
           android:name="android.intent.action.USER_PRESENT" />
       </intent-filter>
    </receiver>
    <provider
      android:name="com.google.android.apps.testapp.dataaccess.persistence.ContentProvider"
      android:authorities="com.google.android.apps.testapp"
      android:exported="false" />
    <activity android:name="com.google.android.apps.testapp2.ui.home.HomeActivity" android:label="@string/app_name" >
      <intent-filter>
        <action android:name="android.intent.action.MAIN" />
        <category android:name="android.intent.category.LAUNCHER" />
      </intent-filter>
    </activity>
    <activity android:name="com.google.android.apps.testapp2.TestActivity2" />
    <activity android:name="com.google.android.apps.testapp2.PreviewActivity" />
    <activity android:name="com.google.android.apps.testapp2.ShowTextActivity"
        android:excludeFromRecents="true" />
    <activity android:name="com.google.android.apps.testapp2.ShowStringListActivity"
              android:excludeFromRecents="true"
              android:parentActivityName="com.google.android.apps.testapp2.ui.home.HomeActivity" >
    </activity>
    <service android:name="com.google.android.apps.testapp2.TestService" >
      <meta-data android:name="param" android:value="value" />
    </service>
    <service android:name="com.google.android.apps.testapp2.nfcevent.NfcEventService" />
    <receiver android:name="com.google.android.apps.testapp2.ConnectivityReceiver"
        android:enabled="false" >
      <intent-filter>
        <action android:name="android.net.conn.CONNECTIVITY_CHANGE" />
      </intent-filter>
    </receiver>
    <activity-alias android:name="com.google.android.apps.testapp2.BarFoo"
      android:targetActivity="com.google.android.apps.testapp2.FooBar" />
    <provider
      android:name="some.package.with.inner.class$AnInnerClass" />
    <provider
      android:name="com.google.android.apps.testapp"
      android:authorities="com.google.android.apps.testapp.com.google.android.apps.testapp"
      android:exported="false" />
    <provider
      android:name="com.google.android.apps.testapp.PlaceHolderProviderName"
      android:authorities="PlaceHolderProviderAuthorities.com.google.android.apps.testapp"
      android:exported="false" />
    <activity
      android:name="activityPrefix.com.google.android.apps.testapp.activitySuffix" >
      <intent-filter>
        <action android:name="actionPrefix.com.google.android.apps.testapp.actionSuffix" />
      </intent-filter>
    </activity>
    <activity android:name="com.google.android.apps.testapp3.ui.home.HomeActivity" android:label="@string/app_name" >
      <intent-filter>
        <action android:name="android.intent.action.MAIN" />
        <category android:name="android.intent.category.LAUNCHER" />
      </intent-filter>
    </activity>
    <activity android:name="com.google.android.apps.testapp3.TestActivity" />
    <service android:name="com.google.android.apps.testapp3.TestService" />
    <receiver android:name="com.google.android.apps.testapp3.ConnectivityReceiver"
              android:enabled="true" >
      <intent-filter>
        <action android:name="android.net.conn.CONNECTIVITY_CHANGER" />
      </intent-filter>
    </receiver>
  </application>
</manifest>
